platformio.ini

```diff
@@ -33,6 +33,7 @@ extra_configs =
 ;default_envs = esp32dev-all
 ;default_envs = esp32dev-rf
 ;default_envs = esp32dev-pilight
+;default_envs = esp32dev-weatherstation
```

Comment:
If the code works with esp8266 and/or arduino would it be possible to add the corresponding environment definitions?